src/routes/+layout.svelte

```diff
@@ -1,5 +1,15 @@
-<script>
+<script lang="ts">
 	import '../app.css';
+	import { onMount } from 'svelte';
+
+	let currentUrl: string;
+	onMount(() => {
+		currentUrl = window.location.href;
+	});
```

Comment:
I'm not sure this would work. I'm not exactly sure what kind of request discord makes to fetch this information, but it's likely that it won't call onMount before returning the meta property.

Instead, I'd get the url from a `+layout.server.ts` file which has [`url.href`](https://developer.mozilla.org/en-US/docs/Web/API/URL):
```ts
export const load: ServerLoad = ({url}) => {
    return { path: url.href };
}
```
You could also do a similar thing using hooks & locals if you think that works better